client/src/components/AddPollDialog.js

```diff
@@ -121,6 +123,12 @@ class AddPollDialog extends Component {
 
     handleSelectChange = e => {
         this.setState({ sendToList: e.target.value });
```

Comment:
As an alternative, you can use custom attribute on MenuItem: `<MenuItem data-title={list.title}>` and then access it as `e.currentTarget.dataset.title`

Reply:
Noted.